ipsec_accel/requirements.txt

```diff
@@ -0,0 +1 @@
+PyYAML
```

Comment:
PyPYAML is already listed in requirements.txt in the main folder of ipu-playbook. We can remove this file